keyboards/pixelspace/shadow80/info.json

```
@@ -0,0 +1,543 @@
+{
+    "keyboard_name": "SHADOW80",
+    "processor": "STM32F103",
+    "bootloader": "stm32duino",
+    "manufacturer": "Here VoLand",
+    "url": "",
+    "maintainer": "Here VoLand @Vem",
+    "usb": {
+        "vid": "0x0011",
+        "pid": "0x0011",
+        "device_version": "0.0.1"
+    },
```

migrated from `rules.mk`
Would advise `command` is disabled when `mousekey` and/or `extrakey` are enabled
```suggestion
    },
    "features": {
        "bootmagic": true,
        "mousekey": true,
        "extrakey": true,
        "nkro": true,
        "rgblight": true
    },
```